### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BroOtti**
Removed the note of not yet released version 2.2.1.0 
Added section "Determine missing translations"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +12,11 @@ Under the 'Strings' object, there is a key for each original text node in Englis
 
 Any strings within AMP that don't exist in your translation file will remain as English, so you can work on it gradually over time.
 
+## Determine missing translations
+
+AMP offers a TranslatorMode. To use it open your browser console (F12) and execute `Locale.SetTranslatorMode()`. While browsing through AMP it will keep track of translatable strings. After your done using AMP for a bit execute `Locale.DownloadJSONFile()` to download your new JSON file.
```
Comment (IceOfWraith) [inline README.md:17]: "After you're done"

File: README.md [hunk 2]
```diff
@@ -14,6 +12,11 @@ Under the 'Strings' object, there is a key for each original text node in Englis
...
+Note that you should enable the TranslatorMode already on the login screen.
```
Comment (IceOfWraith) [inline README.md:18]: "while on"

**Review by IceOfWraith — CHANGES_REQUESTED** [review]
Thanks for the addition! I put two edits to clean up the wording. :)

File: README.md (hunk 1 above)
Comment (BroOtti) [reply README.md]: haha, my bad :D

**Review by IceOfWraith — APPROVED** [review]
Very good! Thank you!

File: README.md (hunk 2 above)
Comment (BroOtti) [reply README.md]: Thanks :)

**Pull request merged**